packages/pwa-kit-extension-sdk/src/express/placeholders/application-extensions.ts

```diff
@@ -6,15 +6,12 @@
  */
 /* istanbul ignore file */
 
-import {ApplicationExtensionConfig as ApplicationExtensionConfigBase} from '../../types'
-import {ApplicationExtension as ApplicationExtensionBase} from '../classes/application-extension'
+// Hey! This file is a magic placeholder.
+// Do not edit this file.
 
-// Types
-// TODO: Move this to types.
-type ExtendedApplicationExtensionConfig = ApplicationExtensionConfigBase
-type ApplicationExtension<T extends ApplicationExtensionConfigBase> = ApplicationExtensionBase<T>
-
-const APPLICATION_EXTENSIONS: ApplicationExtension<ExtendedApplicationExtensionConfig>[] = []
-
-// Export the array
-export const getApplicationExtensions = () => APPLICATION_EXTENSIONS
+// During build process, this file is replaced with the actual implementation.
+// see pwa-kit-extension-sdk/configs/utils.ts for more details.
+export const getApplicationExtensions = () => {
+    console.error('Something went wrong. Cannot load application extensions.')
+    return []
+}
```

Comment:
👍 I like that this makes it look more like the placeholder code it's meant to be.